angular.json

```diff
@@ -29,7 +29,9 @@
               "src/styles.scss",
               "./node_modules/@fortawesome/fontawesome-free/css/all.min.css"
             ],
-            "scripts": []
+            "scripts": [
+              "src/scripts/p3.js"
+            ]
```

Comment:
out of curiosity, what is p3.js? Does the code come from a library?

Reply:
Originally I stole it from Steve's code, but now it looks nothing like it!